lib/encoder.js

```diff
@@ -188,23 +208,38 @@ function defineInstanceMembers() {
   this.decodeBlob = function (bytes) {
     return this.handleBuffer(bytes);
   };
-  this.decodeCustom = function (bytes, typeName) {
+
+  /**
+   * 
+   * @param {Buffer} bytes 
+   * @param {ColumnInfo | string} typeName
+   * @param {ColumnInfo | null} columnInfo 
+   * @returns 
+   */
+  this.decodeCustom = function (bytes, typeName, columnInfo = null) {
```

Comment:
I really dislike the idea of adding an extra param here.

The [original impl](https://github.com/datastax/nodejs-driver/blob/master/lib/encoder.js#L1621) of this function passes in "type.info" from the type info it gets as an arg.  Your [new impl](https://github.com/SiyaoIsHiding/nodejs-driver/blob/vector/lib/encoder.js#L1771) passes in "type.info" as well but also passes in... "type" as well.  One of these args seems redundant; why pass "type.info" when you can just as easily get the "info" property of the type information you're also passing.

Seems like you can do the _exact_ same thing by just passing "columnInfo" here and adding "typeName = columnInfo.info" near the top of this function.